components/navigation/navigation.css

```diff
@@ -0,0 +1,61 @@
+.nav-header {
+    background-color: var(--color-primary);
+    display: flex;
```

Comment:
I suggest you to make a utility class for display:flex as it will help you in your future projects.

Reply:
Sure. Will check this.